content/js/geolocation.md

```diff
@@ -1,68 +1,54 @@
-## Saving a location on record
+## Save a location on record
 
-```js
+``` javascript
```

Comment:
I think there are no spaces before `javascript`
https://help.github.com/articles/creating-and-highlighting-code-blocks/